src/Compilers/CSharp/Test/Symbol/Symbols/RequiredMembersTests.cs

```diff
@@ -4482,35 +4663,6 @@ public record C
         );
     }
 
-    [Fact, CompilerTrait(CompilerFeature.NullableReferenceTypes)]
-    public void RequiredMemberSuppressesNullabilityWarnings_ChainedConstructor_11()
```

Comment:
As in the previous PR, this test was moved, not deleted, to be with the rest of the `RequiredMemberSuppressesNullabilityWarnings_ChainedConstructor_` tests.